ios/StoresTests/StoreTests.swift

```diff
@@ -59,6 +59,16 @@ class StoreTests: XCTestCase {
 
         XCTAssertEqual(obj, stores.get(.codableKey))
     }
+    
+    func testUserDefaultArrayBasedKey() {
+        let expected = [
+            ExampleStruct(name: "User 1", age: 11),
+            ExampleStruct(name: "User 2", age: 22)
+        ]
```

Comment:
Nit: The style we typically go with is to put a new line after multi line statements.

Reply:
would that be like this?
```suggestion
        ]

```

Reply:
Perfect, thanks